enumerators.c

```diff
@@ -0,0 +1,21 @@
+enum boolean
+{
+ false ,true;
+}
+boolean iseven(int x);
+  return (true);
+else
+  return(false);
+Void main()
```

Comment:
Did you run this program? Please run the program once and see if its working. Like, take inputs and see whether the output is as per your expectations.